.github/PULL_REQUEST_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,74 @@
+---
+name: New component
+about: Outline the requirements for a new component.
+labels: feature
+---
+
+## New component
+
+A brand new web component.
+
+
+### Related issue
+
+- (#100) Description
+
+
+### Preview
+
+<!-- Suggest linking to Netlify or a public sandbox; not a resource behind a log-in or VPN -->
+Link(s) to demo page(s) where this element can be viewed:
+- [Link](https://5e6089f7c8e38b0008963801--happy-galileo-ea79c4.netlify.com/examples/) 
+
+
+### What has changed and why
+
+<!-- See the `DISCOVERY.md` template at the root of the project; this can be copied and stored in your component folder for reference. -->
+
+Link to discovery documentation (should contain all the planning and requirements for this work):
+   - 
+
+
+### Testing instructions
+
+<!-- Be sure to include detailed instructions on how your update can be tested by another developer. -->
+
+1. 
+
+
+#### Browser requirements
+
+Your component should work in all of the following environments:
+
+- [ ] Latest 2 versions of Edge
+- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
+- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
+- [ ] Firefox 60.7.2 (or latest version for Red Hat Enterprise Linux distribution)
+- [ ] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
+- [ ] Latest 2 versions of Safari
+- [ ] Galaxy S9 Firefox
+- [ ] iPhone X Safari
```

Comment:
could we simply these mobile checkboxes to "Android mobile device" / "Apple mobile device"

Reply:
Sure!